src/main/java/tutorlink/listpackage/StudentList.java

```diff
@@ -1,32 +1,35 @@
 package tutorlink.listpackage;
 
 import tutorlink.exceptionspackage.ItemNotFoundException;
-import tutorlink.studentpackage.StudentClass;
+import tutorlink.exceptionspackage.TutorLinkException;
+import tutorlink.studentpackage.Student;
 
 import java.util.ArrayList;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 public class StudentList extends ItemList {
-    private ArrayList<StudentClass> studentArrayList;
+    private ArrayList<Student> studentArrayList;
 
     public StudentList() {
         this.studentArrayList = new ArrayList<>();
     }
 
-    public boolean deleteStudent(StudentClass student) {
+    public boolean deleteStudent(String matricNumber, String name) {
+        Student student = new Student(matricNumber, name);
         return studentArrayList.remove(student);
     }
 
-    public void addStudent(StudentClass student) {
+    public void addStudent(String matricNumber, String name) {
```

Comment:
This should be where duplicate student be checked instead of other command/method

Reply:
Ok added